lib/navigation/preprocess_imu.cpp

```diff
@@ -2,55 +2,99 @@
 
 namespace hyped::navigation {
 
-ImuPreprocessor::ImuPreprocessor()
+ImuPreprocessor::ImuPreprocessor(core::ILogger &logger)
+    : logger_(logger),
+      num_outliers_per_imu_({0, 0, 0, 0}),
+      are_imus_reliable_({true, true, true, true}),
+      num_reliable_accelerometers_(core::kNumImus)
```

Comment:
is there a rule for when imu vs accelerometer is used? here I feel like it should be either `are_imus_reliable_` and `num_reliable_imus_` or both accelerometer

Reply:
@BenET01 think sensors prefer accelerometer so stick to that for the sake of naming consistency?

Reply:
+1 on accelerometer